protocol/app/app.go

```diff
@@ -1594,15 +1593,14 @@ func (app *App) initOracle(pricesTxDecoder process.UpdateMarketPriceTxDecoder) {
 			compression.NewDefaultVoteExtensionCodec(),
 			compression.NewZLibCompressor(),
 		),
-		// We are not using the slinky PreBlocker, so there is no need to pass in PreBlocker here for
+		// We are not using the slinky PreBlocker/PriceApplier, so there is no need to pass in PreBlocker here for
```

Comment:
Are the `PreBlocker` comments still relevant after this upgrade?

Reply:
Probably not since we've moved away from using the preblocker here and have a dedicated interface now instead. Will remove.